source/includes/documentation/_vue_js_elements.md

```diff
@@ -2996,6 +2996,22 @@ For examples of usage see: [JDI Vuetify TreeView tests](https://github.com/jdi-t
 ### 5.37 Button (VuetifyButton)
 
 ```java
+//@FindBy(css = ".elevation-2")
+@UI(".elevation-2")
+public static VuetifyButton commonButton;
+
+@Test(description = "Test checks that common button is clickable")
+public void commonButtonsTests() {
+  commonButton.has().hasNoLabel();
+  commonButton.show();
+  commonButton.is().displayed();
+  commonButton.has().elevated();
+  commonButton.has().lightTheme();
+  commonButton.click();
+  commonButtonState.has().text("Button clicked");
+  }
+  
+  
```

Comment:
Please, update the test for Depressed Buttons. It was refactored.

Reply:
Thank you for keeping me informed! Done.